spatialprofilingtoolbox/ondemand/providers/counts_provider.py

```diff
@@ -0,0 +1,103 @@
+"""Count cells for a specific signature, over the specially-created binary-format index."""
```

Comment:
Hm too bad this was not `git mv`-ed? There is no git diff, it appears as new code.

Reply:
I moved it into the provider subdirectory so it shows up as new. I also excised several functions that overlapped with `ProximityProvider`.

Reply:
I think if you use `git mv` instead of `mv` then git diff will work correctly, like:
```txt
diff --git a/example.md b/example2.md
similarity index 99%
rename from example.md
rename to example2.md
index cfb57be..bac8b60 100644
--- a/example.md
+++ b/example2.md
@@ -1,6 +1,8 @@
 
+New text....
+

```

Reply:
But perhaps this only works if you move the file before editing.

Reply:
If I recall correctly, I created a new copy of the file (edited), committed it, and then deleted the old files in a separate commit? Unsure. I think this usually gets picked up correctly as I've seen it before and I've never used `git mv` in my life. It could also be that the changes were so different between the copied, edited version and the original that it didn't get picked up.

Reply:
Gotcha, that makes sense. I believe I have also experienced git intelligently figuring out that a file has been renamed without explicitly being told about it. After I noticed it fail to do so sometimes, I changed to always using `git mv` for files under source control.